### DISCUSSION TITLE
Alerts are lost

### DISCUSSION THREAD

**Discussion by molixiaoge** [ccfos/nightingale#1292] [Q&A]
自定义了一个脚本执行频率60秒，监控日志文件里面error关键字，根据和上一次的error数量进行比较 返回error数量差异 ret值，后端添加监控语句checkexception_exec_ret!=0 执行频率15秒 持续时长30秒。在这样的场景下会丢失一次警告，如第一次ret=0 第二次ret=2 第三次ret=3 第四次ret=0，按理来说应该警告2次 一次为ret=2 一次为ret=3 但是只接收到了一次警告ret=2，ret=3的情况没有任何警告产生。

前端版本：5.3.0
后端版本：5.7.0

后台数据如下:
![图片](https://user-images.githubusercontent.com/1323533/204455224-e500936e-f194-49b8-849f-924a19b97fdb.png)

![图片](https://user-images.githubusercontent.com/1323533/204454861-e90452a9-7273-4b9b-b41b-b12e2beed1e8.png)

监控机器日志信息如下:
![图片](https://user-images.githubusercontent.com/1323533/204454978-48c93510-47c0-45c6-981f-fe0081f8f5de.png)

**Comment by 710leo** [comment]
<img width="553" alt="image" src="https://user-images.githubusercontent.com/2169271/204467483-0e02f262-cb25-4259-9537-5c878ccf9989.png">
@molixiaoge 可以关注下这个配置

**Comment by molixiaoge** [comment]
监控机器配置如下

![图片](https://user-images.githubusercontent.com/1323533/204479913-6f9dd6bc-904a-4b45-bec3-c19f668e9a8f.png)

![图片](https://user-images.githubusercontent.com/1323533/204479874-e7a177e4-38c6-4c67-98c9-d9cf02b48013.png)

后台告警规则配置测试结果如下:
执行频率15秒 持续时长30秒 重复发送频率0分钟  结果:警告丢失
执行频率15秒 持续时长15秒 重复发送频率0分钟  结果:警告丢失
执行频率15秒 持续时长30秒 重复发送频率1分钟  结果:警告丢失
执行频率15秒 持续时长15秒 重复发送频率1分钟  结果:警告不丢失了

目前看来是这个重复发送频率导致的？那么执行频率 持续时长 重复发送频率和机器的采集频率有什么关系呢？
